requirements.in

```diff
@@ -1,11 +1,11 @@
-Django<3.3                        # core server-side framework
+Django==4.2.13                    # core server-side framework
```

Comment:
Its really best to be as vague in the requirements.in file as safely possible. We know we want to stay on 4.2.X due to long term support, but would we really not want to update to 4.2.14 if a security patch was released? `<4.2` is more likely to be the behavior we want

Reply:
riiight we definitely want the `<4.3` to get the most recent 4.2.X patch!